i18n/fr/docusaurus-plugin-content-docs/current/additional.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 3000
+---
+
+# 🛸 Ressources supplémentaires
+
+## Ressources
+
+Une vidéo d'introduction fantastique à PE : https://youtube.com/watch?v=dOxUroR57xs&feature=shares
+
+Un très bon guide d'ingénierie de prompts courts : https://help.openai.com/en/articles/6654000-best-practices-for-prompt-engineering-with-openai-api
+
+Une excellente introduction à l'ingénierie de prompts : https://humanloop.com/blog/prompt-engineering-101
+
+Une belle collection de documents d'ingénierie de prompts : https://github.com/dair-ai/Prompt-Engineering-Guide
+
+De nombreux autres documents d'ingénierie de prompts : https://github.com/thunlp/PromptPapers
+
+Cours CMU sur l'ingénierie de prompts : https://youtu.be/5ef83Wljm-M
+
+Comment fonctionne Copilot : https://thakkarparth007.github.io/copilot-explorer/posts/copilot-internals.html
+
+Un guide de débutants pour l'écriture de prompts de Zapier : https://zapier.com/blog/gpt-3-prompt/
+
+Une liste de ressources impressionnante sur l'ingénierie des prompts de A à Z : https://github.com/promptslab/Awesome-Prompt-Engineering
+
+
+### Exemples / Idées de prompts GPT-3
+
+https://sharegpt.com 
+
+https://www.learngpt.com 
+
+https://chatgpt.getlaunchlist.com 
+
+https://prompts.chat
+
+
+## Faits
+
+GPT-3 *n'est* PAS déterministe : https://twitter.com/BorisMPower/status/1608522707372740609
+
+## Personnes à suivre
+
+Je trouve du contenu significatif auprès de ces personnes.
+
+
+[@chillzaza_](https://mobile.twitter.com/chillzaza_) Ingénierie de prompts, outils, toolbot
+
+[@mathemagic1an](https://mobile.twitter.com/mathemagic1an) Diverses incitations, informations sur PE, GPT3
+
+[@goodside](https://twitter.com/goodside/status/1588247865503010816) Injection de prompts, outillage PE
+
+[@hwchase17](https://twitter.com/hwchase17) Dév principal de Langchain
+
+[@omarsar0](https://twitter.com/omarsar0) Lead DAIR AI
+
+
+Veuillez également vérifier les comptes que je suis : https://twitter.com/learnprompting/following 
```

Comment:
The original English was unfortunately "check" but actually means to say "check out":

```suggestion
Voir aussi les comptes que je suis : https://twitter.com/learnprompting/following 
```

"`Veuillez également ...`" sounds way too formal to my ears haha - and the original English didn't say "Please"